input/pagecontent/reference.md

```diff
@@ -843,7 +845,7 @@ A number of rules may follow the keyword statements. The grammar and meaning of
 
 #### Defining Aliases
 
-Aliases allow the user to replace a lengthy url, oid, or uuid with a short string. Aliases are for readability only, and do not change the meaning of rules. Typical uses of aliases are to represent code systems and canonical URLs.
+Aliases MAY be used to replace a lengthy url, oid, or uuid with a short string. Aliases are for readability only, and SHALL not change the meaning of rules. Typical uses of aliases are to represent code systems and canonical URLs.
```

Comment:
Does the "not" after "SHALL" need to be all-caps here?

Reply:
Indeed! Fixed and rebased into 1acd751.